HumbleBundleBotRegistration/Pages/Index.cshtml

```diff
@@ -0,0 +1,37 @@
+﻿@page "/"
+
+<h1>HumbleBundleBot</h1>
```

Comment:
It would be interesting to see if we can pull this info automatically from the Readme

Reply:
I agree, maybe create an issue for it?